tests/suite/test_auth_basic_policies.py

```diff
@@ -49,7 +49,6 @@
 def to_base64(b64_string):
     return b64encode(b64_string.encode("ascii")).decode("ascii")
 
-
```

Comment:
I had put a pytest mark here when running these tests. Will re-add this line to reduce changes.